includes/class-wc-cart-totals.php

```diff
@@ -313,6 +313,8 @@ protected function get_fees_from_cart() {
 				}
 			}
 
+			$fee->taxes = apply_filters( 'woocommerce_get_fees_from_cart_taxes', $fee->taxes, $fee, $this );
```

Comment:
I'd rename this for clarity since the method name is not unique.

`woocommerce_cart_totals_get_fees_from_cart_taxes`

The other actions and filters don't have this prefix; they existed before cart-totals class was added.

Reply:
Of course, I agree.
You can rename it to:
woocommerce_cart_totals_get_fees_from_cart_taxes